src/cephadm/cephadm

```diff
@@ -16,27 +16,8 @@ DEFAULT_TIMEOUT = None  # in seconds
 DEFAULT_RETRY = 10
 SHELL_DEFAULT_CONF = '/etc/ceph/ceph.conf'
 SHELL_DEFAULT_KEYRING = '/etc/ceph/ceph.client.admin.keyring'
+CEPHADM_TOOL_BASE_PATH = '/var/lib/ceph/%s' # Variable part is for the cluster fsid
```

Comment:
It's a small nit, but I'd refrain from using constants that are not, well, _constant_.

Reply:
Also, isn't the base path something like:

`/var/lib/ceph/$fsid/cephadm/` ?

Reply:
changed